test/test_smallbaselineApp.py

```diff
@@ -34,7 +32,9 @@
     'https://zenodo.org/record/3952917/files/KujuAlosAT422F650.tar.xz',
 ]
 
-PROJ_NAME_LIST = [os.path.basename(url).split('.tar.xz')[0] for url in URL_LIST]
+PROJ_NAMES = [os.path.basename(url).split('.tar.xz')[0] for url in URL_LIST]
+TEMPLATE_FILES = [os.path.join(os.path.dirname(__file__), 'configs/{}.txt'.format(proj_name))
+                  for proj_name in PROJ_NAMES]
```

Comment:
```suggestion
TEMPLATE_FILES = [os.path.join(os.path.dirname(os.path.abspath(__file__)), 'configs/{}.txt'.format(proj_name))
                  for proj_name in PROJ_NAMES]
```

I checked out your branch and tried a
```sh
python test/test_smallbaselineApp.py
```
which failed with
```
FileNotFoundError: test/configs/FernandinaSenDT128.txt
```
Adding the above suggested change, however, made it work.

Suprisingly, in this case `__file__` isn't an absolute path, which it needs to be since [`smallBaselineApp.py` is called](https://github.com/insarlab/MintPy/pull/653/files#diff-96f2c9926af3120f234f57b879cf7c88579654d960e4cbf9d42d0fb688a827c2R148) from [the `work_dir`](https://github.com/insarlab/MintPy/pull/653/files#diff-96f2c9926af3120f234f57b879cf7c88579654d960e4cbf9d42d0fb688a827c2R128-R129)

---
You could also make it more compact with:
```python
from pathlib import Path
TEMPLATE_FILES = [Path(__file__).resolve().parent / 'configs' / f'{proj_name}.txt'
                  for proj_name in PROJ_NAMES]
```

Reply:
Thank you for the suggestion, I have added it.